web/src/app/components/elements/Tables.tsx

```diff
@@ -84,7 +85,61 @@ const StStandardTable = styled.table`
         tr {
             border-bottom: 1px solid  #DCDCDC;
             :hover {
-                background-color: ${EColor.LIGHT_GREEN};
+                background-color: ${(props: IStTable) => props.disableHover ? "" : EColor.LIGHT_GREEN};
+                cursor: ${(props: IStTable) => props.disableHover ? "" : "pointer"};
+            }
+        }
+        td {
+            padding: 12px 10px;
+            font-size: 14px;
+        }
+    }
+`;
+
+interface ISimpleTable {
+    rows: IStandardTableRow[];
+    selectRow?: Function;
+    disableHover?: boolean;
+}
+
+export const SimpleTable = (props: ISimpleTable) => {
+    return (
+        <StSimpleTable disableHover={props.disableHover}>
+            <tbody>
+                {props.rows.map((row, index) => {
+                    return (
+                        <tr key={index}
+                            onClick={typeof props.selectRow !== "undefined" ? () => {
+                                if (props.selectRow) {
```

Comment:
unnecessary `if` now that we have a check outside the function. 

```javascript
typeof props.selectRow !== "undefined" ? () => props.selectRow(row): undefined
```

Reply:
That is what we talked about that does not work... typescript still thinks it can be undefined